app/src/tingapp.js

```diff
@@ -14,7 +14,13 @@ class Tingapp {
 
         const newDocumentPath = path.join(tempDir, 'untitled.tingapp');
         console.log(newDocumentPath);
-        fsextra.copySync('./default.tingapp', newDocumentPath);
+        // check for dev to select where to fetch resources from
+        if(/[\\/]electron-prebuilt[\\/]/.test(process.execPath)){
+          fsextra.copySync('./app/default.tingapp', newDocumentPath);
+        }else{
+          fsextra.copySync(path.join(process.resourcesPath,'app.asar','default.tingapp'), newDocumentPath);
+        }
```

Comment:
Actually this would not work (after retesting) due to the fact we utilise electron-prebuilt for development work see: [electron-is-dev](https://github.com/sindresorhus/electron-is-dev)